data/test/scenarios/test_move_unit.cfg

```diff
@@ -258,6 +267,15 @@
     [/event]
 )}
 
+#####
+# API(s) being tested: [move_unit]
+##
+# Actions:
+# Move alice in a circle.
```

Comment:
Move alice in a circle, returning to his starting hex in a single move.

Explanation that could be added before the last ASSERT in this test:
        # The assert can't check that Alice moved, but it does check that the implementation of
        # [move_unit] didn't choose an unoccupied hex next to the (initially-occupied) starting hex.